docs/static/monitoring/monitoring-ea.asciidoc

```diff
@@ -0,0 +1,100 @@
+[[monitoring-with-elastic-agent]]
+=== Collect {ls} monitoring data with {agent}
+[subs="attributes"]
+++++
+<titleabbrev>{agent} collection</titleabbrev>
+++++
+
+You can use {agent} to collect data about {ls} and ship it to the monitoring
+cluster. The benefit of {agent} collection is that the monitoring agent remains
+active even if the {ls} instance does not. Plus you can mange all your
```

Comment:
```suggestion
active even if the {ls} instance does not. Plus you can manage all your
```

Reply:
Ha! Good catch. This is my go-to typo. I'm sure there are still docs at Big Blue that talk about a "Metadata Manger"